cmd/supervisor/supervisor.go

```diff
@@ -50,11 +59,17 @@ func main() {
 	}
 
 	if *svflags.FlagInstall {
+		ensureInstallFlags()
 		path, err := thisPath()
 		if err != nil {
 			log.Fatal().Err(err).Msg("Failed to determine supervisor's path")
 		}
-
+		cfg := config.Config{
+			Uid: *svflags.FlagUid,
+		}
+		if err := cfg.Write(); err != nil {
```

Comment:
What about removing config during uninstall?

Reply:
As a common practice, configs are not uninstalled.